src/common/drpc.c

```diff
@@ -45,7 +45,7 @@ daos_drpc_alloc(void *arg, size_t size)
 	struct drpc_alloc *alloc = arg;
 	void *buf;
 
-	D_ALLOC(buf, size);
+	buf = malloc(size);
```

Comment:
The whole reason this callback exists is to make use of D_ALLOC() to allow for fault injection testing.  I put quite a lot of effort into this to make it work and catch all the locations where the allocator needs to be provided.

Reply:
I see now. So, replaced with original macro with the same functionality but useing system calloc/free.